src/PointwiseFunctions/ScalarTensor/ConstraintGammas.hpp

```diff
@@ -0,0 +1,98 @@
+// Distributed under the MIT License.
```

Comment:
Could you add a few tests for the different features being added?
1. Can you use `TestHelpers::db::test_simple_tag` for the simple tags?
2. Can you use `TestHelpers::db::test_compute_tag` for the compute tags?
3. Can you use `TestHelpers::test_option_tag` to test the option tag?
4. Can you add the compute tags to a DataBox and just make sure you can retrieve things correctly? You could use a trivial FunctionOfTime and a trivial damping function (just a constant).